scipy/stats/_bootstrap.py

```diff
@@ -128,13 +128,30 @@ def _bootstrap_iv(data, statistic, vectorized, axis, confidence_level,
     data_iv = []
     for sample in data:
         sample = np.atleast_1d(sample)
-        if sample.shape[axis] <= 1:
+        if sample.shape[axis_int] <= 1:
             raise ValueError("each sample in `data` must contain two or more "
                              "observations along `axis`.")
         sample = np.moveaxis(sample, axis_int, -1)
         data_iv.append(sample)
 
-    # should we try: statistic(data, axis=axis) here?
+    if not isinstance(paired, bool):
```

Comment:
As per @WarrenWeckesser comment on the other PR, this could check instead for True False.

Reply:
That is what I had originally in the other PR; I changed it per your comment. Somehow I missed the discussion about the original being preferable. OK, I'll change them.

Reply:
Sorry about the change here!